app/controllers/user_friends_controller.rb

```diff
@@ -1,4 +1,14 @@
 class UserFriendsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    if params[:id] != nil
```

Comment:
Use the return of the conditional for variable assignment and comparison.<br>Prefer `!expression.nil?` over `expression != nil`.